### PR TITLE
feat: update to show installation requirements

### PR DESCRIPTION
**Pull request by dtarnawsky**
This adds the missing requirements for minimum iOS deployment target of 14.0 and the skipLibCheck property in TSConfig.

### CODE REVIEW

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
The original file is the one inside plugin folder, then it gets copied into the root if running `npm run build`.
https://github.com/ionic-team/capacitor-google-maps/blob/main/plugin/README.md

The deployment target should also be changed in the project, not just in the Podfile, the text might look like this 
https://capacitorjs.com/docs/v5/updating/4-0#raise-ios-deployment-target

**Comment by dtarnawsky** [comment]
It says:

> Additionally, you will need to open your project in XCode and set the `iOS Deployment Target` to `iOS 14.0` or higher in the `Build Settings` tab.

**Comment by jcesarmobile** [comment]
Yeah, that's incomplete or unclear, you have to change it for both the project and all the targets, each one of them have a `Build settings` tab with independent deployment target values and has to be changed for all.

Captura de pantalla 2024-09-24 a las 17 03 12

**Comment by dtarnawsky** [comment]
I've cleaned up the wording to specify that you need to do it for the target app. The projects deployment target doesnt need to be updated (see this [sample app](https://github.com/dtarnawsky/cs-google-maps))

**Review by jcesarmobile — CHANGES_REQUESTED** [review]
Despite the app might build with different deployment targets on the project and the targets, it can cause issues, so it's better to always have the versions in sync.

The new text is incorrect, it should say in the opposite order, like 
> select `Targets` > `App` (or the name of your target) and in the `Build Settings` tab
Users could have more than a target, so that's why I'm asking to use the same text we have on the migrate guide, and also for consistency

**Comment by dtarnawsky** [comment]
Ok, changed again.

File: plugin/README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Version 6 of this plugin has a minimum deployment target of iOS 14.0. You will n
 platform :ios, '14.0'
 ```
 
-Additionally, you will need to open your project in XCode and in the `Build Settings` tab select `Targets` > `App` (or the name of your target) and set the `iOS Deployment Target` to `iOS 14` or higher.
+Additionally, you will need to open your project in XCode and in the `Build Settings` tab for your `Project` and for each `Target` set the `iOS Deployment Target` to `iOS 14` or higher.
```
Comment (jcesarmobile) [inline plugin/README.md:31]:
```suggestion
Additionally, you will need to open your project in XCode and in the `Build Settings` tab for your `Project` and for each `Target` set the `iOS Deployment Target` to `iOS 14.0` or higher.
```
Adding the `.0` as `14` is not a valid value

**Review by jcesarmobile — APPROVED** [review]

**Pull request merged**